client/settings/fraud-protection/advanced-settings/cards/order-items-threshold.tsx

```diff
@@ -56,15 +58,21 @@ const OrderItemsThresholdCustomForm: React.FC< OrderItemsThresholdCustomFormProp
 	);
 
 	useEffect( () => {
-		settingUI.min_items = minItemsCount;
-		settingUI.max_items = maxItemsCount;
+		settingUI.min_items = minItemsCount
+			? parseInt( minItemsCount + '', 10 )
+			: minItemsCount;
+		settingUI.max_items = maxItemsCount
+			? parseInt( maxItemsCount + '', 10 )
+			: maxItemsCount;
 		setProtectionSettingsUI( protectionSettingsUI );
+		setProtectionSettingsChanged( ( prev ) => ! prev );
```

Comment:
Could we make it `true` instead of `( prev ) => ! prev`? It will toggle between `true` and `false` when I change more than one field in the form.

Reply:
Here we want the `settingsChanged` to be recalculated every time the form changes in the above layer, and triggering changes to ProtectionSettingsUI isn't caught by useEffect in the above layer (as you mentioned on Slack). So it needs to be this way temporarily, until you prepare the PR that fixes the immutability issue.